File: Submissions/GoodBye_2015/A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  int count;
  cin >> count;

  string s1, s2;
  cin >> s1 >> s2;

  if (s2 == "week") {
    if (count == 5 || count == 6) {
      cout << 53 << endl;
    } else {
      cout << 52 << endl;
    }
  } else {
    if (count == 31) {
      cout << 7 << endl;
    } else if (count == 30) {
      cout << 11 << endl; 
    } else {
      cout << 12 << endl; 
    }
  }

}

File: Submissions/GoodBye_2015/A/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);
    int count = sc.nextInt();
    String s1 = sc.next();
    String s2 = sc.next();
    sc.close();
    
    if (s2.equals("week")) {
      if (count == 5 || count == 6) {
        System.out.println(53);
      } else {
        System.out.println(52);
      }
    } else {
      if (count == 31) {
        System.out.println(7);
      } else if (count == 30) {
        System.out.println(11); 
      } else {
        System.out.println(12); 
      }
    }

  }

}